layouts/partials/utils/title.html

```diff
@@ -1,27 +1,30 @@
+{{- $ := index . "$" -}}
+{{- $rawTitle := .title -}}
```

Comment:
It's rather non-obvious now what this template returns, with the final calculations being performed in the return statement. How about making things less confusing and naming this variable `$htmlTitle`? $rawTitle can then be defined afterwards as `|plainify|htmlUnescape` version of it. And `$title` would be based on `$rawTitle` again. Or was allowing HTML tags in `.Site.Title` intentional?